Examples/ArcGISToolkitExamples/TimeSliderExample.swift

```diff
@@ -0,0 +1,91 @@
+//
+// Copyright 2018 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGISToolkit
+import ArcGIS
+
+class TimeSliderExample: MapViewController {
+    
+    private var map : AGSMap?
+    private var timeSlider : TimeSlider?
```

Comment:
I would suggest to explicitly init the TimeSlider from line 32 here, there seems to be no need for the timeSlider property to be optional.